modules/pages/client/components/Team.component.js

```diff
@@ -0,0 +1,497 @@
+                >
+                  <a href="https://twitter.com/simison">
+                    <i className="text-muted icon-fw icon-lg icon-twitter"></i>{' '}
+                  </a>
+                </Tooltip>
+                <Tooltip
+                  id="github-tooltip"
+                  tooltip="GitHub"
+                  placement="bottom"
+                >
+                  <a href="https://github.com/simison">
+                    <i className="text-muted icon-fw icon-lg icon-github"></i>{' '}
+                  </a>
+                </Tooltip>
+                <Tooltip
+                  id="linkedin-tooltip"
+                  tooltip="LinkedIn"
+                  placement="bottom"
+                >
+                  <a href="https://www.linkedin.com/in/mikaelkorpela">
+                    <i className="text-muted icon-fw icon-lg icon-linkedin"></i>{' '}
+                  </a>
+                </Tooltip>
+              </p>
+            </div>
+          </div>
+          <br />
+          <br />
+        </div>
+        <div className="col-xs-12 col-sm-4">
+          {/* Natalia */}
+          <div className="media">
+            <img
+              className="media-object img-circle pull-left"
+              src="/img/team-natalia.jpg"
+              width="100"
+              alt="Natalia"
+            />
+            <div className="media-body">
+              <h4 className="media-heading">Natalia</h4>
+              <p className="text-color-links">
+                {t(
+                  'Coming from a multinational business background, Natalia is an enthusiastic of the share and gift economy. Veteran globetrotter and volunteer for different NGOs from hospex as Couchsurfing to nonprofit lobbying orgs.',
+                )}
+              </p>
+              <p>
+                <Tooltip
+                  id="trustroot-tooltip"
+                  tooltip="Trustroots"
+                  placement="bottom"
+                >
+                  <a href="/profile/natalia_sevilla">
+                    <i className="text-muted icon-fw icon-lg icon-tree"></i>{' '}
+                  </a>
+                </Tooltip>
+                <Tooltip
+                  id="linkedin-tooltip"
+                  tooltip="LinkedIn"
+                  placement="bottom"
+                >
+                  <a href="https://www.linkedin.com/in/natalia-s%C3%A1enz-alban%C3%A9s-38469227/">
+                    <i className="text-muted icon-fw icon-lg icon-linkedin"></i>{' '}
+                  </a>
+                </Tooltip>
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-xs-12 col-sm-4">
+          {/* Kasper */}
+          <div className="media">
+            <a className="pull-left" href="https://guaka.org/">
+              <img
+                className="media-object img-circle"
+                src="//gravatar.com/avatar/edccc1f234833b3b5c266c18c2db139d?s=200"
```

Comment:
Kaspers photo is broken, same on production, but maybe we can find a fresh one :)

Reply:
Here is a photo of him https://www.crunchbase.com/person/kasper-souren :)

Or this nice one from 15 years ago https://sw.m.wikipedia.org/wiki/Picha:Kasper_Souren_Berlin_Beach_Camp_september_2005.jpg

Reply:
It's broken in prod too, so can leave it for the unknown future :)